lib/Runtime/Library/JavascriptBigInt.cpp

```diff
@@ -12,14 +12,21 @@ namespace Js
     }
 
     JavascriptBigInt * JavascriptBigInt::CreateZero(ScriptContext * scriptContext)
+    {
+        return JavascriptBigInt::CreateZeroWithLength(1, scriptContext);
+    }
+
+    JavascriptBigInt * JavascriptBigInt::CreateZeroWithLength(digit_t length, ScriptContext * scriptContext)
     {
         JavascriptBigInt * bigintNew = RecyclerNew(scriptContext->GetRecycler(), JavascriptBigInt, scriptContext->GetLibrary()->GetBigIntTypeStatic());
-        bigintNew->m_length = 1;
+        bigintNew->m_length = length;
         bigintNew->m_isNegative = false;
-        bigintNew->m_maxLength = 1;
+        bigintNew->m_maxLength = length;
         bigintNew->m_digits = RecyclerNewArrayLeaf(scriptContext->GetRecycler(), digit_t, bigintNew->m_length);
-        bigintNew->m_digits[0] = 0;
-        
+        for (digit_t i = 0; i < length; i++)
```

Comment:
You can avoid needing this loop if you use `RecyclerNewArrayLeafZ` to do a zero-filled allocation

Reply:
fixed. Thanks, I always forget this macro.

---
In reply to: [234425920](https://github.com/Microsoft/ChakraCore/pull/5835#discussion_r234425920) [](ancestors = 234425920)